src/overlays/ovl_i6/fox_ve2.c

```diff
@@ -0,0 +1,4132 @@
+            Audio_PlaySfx(0x3103108B, &gActors[10].sfxPos, 4, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+            Audio_PlaySfx(0x3103108B, &gActors[11].sfxPos, 4, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+        }
+        gActors[10].obj.pos.x = boss->obj.pos.x + 200.0f;
+        gActors[10].obj.pos.y = boss->obj.pos.y - 200.0f;
+        gActors[10].obj.pos.z = boss->obj.pos.z + 200.0f;
+        gActors[10].unk_0B8 = 0;
+        gActors[10].timer_0C2 = 20;
+        gActors[10].info.unk_1C = 0.0f;
+        gActors[11].obj.pos.x = boss->obj.pos.x - 200.0f;
+        gActors[11].obj.pos.y = boss->obj.pos.y - 200.0f;
+        gActors[11].obj.pos.z = boss->obj.pos.z + 200.0f;
+        gActors[11].unk_0B8 = 0;
+        gActors[11].timer_0C2 = 20;
+        gActors[11].info.unk_1C = 0.0f;
+    }
+    Math_SmoothStepToF(&boss->fwork[21], boss->fwork[22], 1.0f, 6.0f, 0);
+    Math_SmoothStepToF(&boss->fwork[23], boss->fwork[24], 0.3f, 0.01f, 0);
+}
+
+s32 func_i6_8018B47C(s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3f* rot, void* data) {
+    Boss* boss = (Boss*) data;
+
+    if (boss->fwork[21] >= 254.0f) {
+        if (limbIndex == 2) {
+            RCP_SetupDL(&gMasterDisp, 0x16);
+            GPC((s32) boss->fwork[25], 255, 255, 255);
+        } else {
+            RCP_SetupDL(&gMasterDisp, 0x17);
+        }
+    } else {
+        RCP_SetupDL(&gMasterDisp, 0x33);
+        GPC(255, 255, 255, (s32) boss->fwork[21]);
+    }
+    if ((limbIndex == 2) && (boss->swork[6] != 0)) {
+        *dList = NULL;
+        return 0;
+    }
+    if ((limbIndex == 1) || (limbIndex == 2)) {
+        Matrix_Translate(gCalcMatrix, pos->x, pos->y, pos->z, 1);
+        Matrix_RotateZ(gCalcMatrix, rot->z * M_DTOR, 1);
+        Matrix_RotateY(gCalcMatrix, rot->y * M_DTOR, 1);
+        Matrix_RotateX(gCalcMatrix, rot->x * M_DTOR, 1);
+        if (*dList != NULL) {
+            Matrix_Push(&gGfxMatrix);
+            Matrix_Push(&gCalcMatrix);
+            Matrix_Mult(gGfxMatrix, gCalcMatrix, 1);
+            if (limbIndex == 1) {
+                Matrix_RotateY(gGfxMatrix, -boss->fwork[14] * M_DTOR, 1);
+                Matrix_Scale(gGfxMatrix, boss->fwork[16] + 1.0f, 1.0f, 1.0f - boss->fwork[16], 1);
+                Matrix_RotateY(gGfxMatrix, boss->fwork[14] * M_DTOR, 1);
+            } else {
+                Matrix_RotateY(gGfxMatrix, boss->fwork[14] * 1.5f * M_DTOR, 1);
+                Matrix_Scale(gGfxMatrix, 2.0f * boss->fwork[16] + 1.0f, 1.0f, 1.0f - 2.0f * boss->fwork[16], 1);
+                Matrix_RotateY(gGfxMatrix, -boss->fwork[14] * 1.5f * M_DTOR, 1);
+            }
+            Matrix_SetGfxMtx(&gMasterDisp);
+            if ((boss->swork[3] & 1) && (boss->fwork[21] >= 254.0f)) {
+                RCP_SetupDL(&gMasterDisp, 0x1B);
+                GPC(255, 0, 0, 255);
+            }
+
+            GDL(*dList);
+            Matrix_Pop(&gGfxMatrix);
+            Matrix_Pop(&gCalcMatrix);
+        }
+        return 1;
+    }
+    return 0;
```

Comment:
```suggestion
        return true;
    }
    return false;
```
OverrideLimbDraws return booleans